test/cli/iGenVar_cli_test.cpp

```diff
@@ -428,7 +428,8 @@ TEST_F(iGenVar_cli_test, test_direct_methods_input)
 {
     cli_test_result result = execute_app("iGenVar",
                                          "-j", data(default_alignment_long_reads_file_path),
-                                         "--method 0 --method 1 --clustering_method 0 --refinement_method 0");
+                                         "--method cigar_string --method split_read "
+                                         "--clustering_method 0 --refinement_method 0");
```

Comment:
Not sure about these two strings. Shouldn't there be a comma in between?

Reply:
No, because its only one argument which is splitted into two lines.